src/components/AnalysisPage/GeoAnalysisBox/GeoAnalysisBox.jsx

```diff
@@ -3,7 +3,7 @@ import { useState, useEffect } from "react";
 import "./geoAnalysisBox.css";
 import StartButton from "./StartButton/StartButton";
 import DisplayStatistic from "../DisplayStatistic";
-import { InfoCircleOutlined } from "@ant-design/icons";
+// import { InfoCircleOutlined } from "@ant-design/icons";
```

Comment:
You might delete this in a later commit but if not; remove this commented out line